ios/FluentUI/Core/Colors.swift

```diff
@@ -465,6 +465,11 @@ public final class Colors: NSObject {
         public static var background: UIColor = background1
     }
 
+    public struct Contact {
+        public static var firstName: UIColor = foreground1
```

Comment:
>foreground1 [](start = 47, length = 11)

your fork might be out of date. These colors were renamed

Reply:
Got it, I'll pull from upstream. Thanks for letting me know!